### PR TITLE
fix CMakelist and some headers for msvc compatibility

### PR DESCRIPTION
**Pull request by tetsurom**
fix CMakelist and some headers for msvc compatibility.

### CODE REVIEW

File: include/minikonoha/minikonoha.h [hunk 1]
```diff
@@ -106,7 +124,11 @@
 #define KLIB    (kctx->klib)->
 
 #define KDEFINE_PACKAGE KonohaPackageHandler
+#ifdef _MSC_VER
+typedef struct KonohaPackageHandlerVar KonohaPackageHandler;
+#else
```
Comment (imasahiro) [inline include/minikonoha/minikonoha.h:129]: why did you remove const attribute from KonohaPackageHandler?

Comment (tetsurom) [inline include/minikonoha/minikonoha.h:129]: It is necessary because C99 style struct initializer is not be used in msvc.

Comment (imasahiro) [inline include/minikonoha/minikonoha.h:129]: OK. I agree.
